package.json

```
@@ -1,7 +1,13 @@
 {
-  "name": "spl-token-wallet",
-  "version": "0.1.0",
-  "private": true,
+  "name": "io.domichain.dwallet",
```

Something more like app name?
```suggestion
  "name": "domichain-wallet",
```